client/components/promo-section/promo-card/index.tsx

```diff
@@ -76,12 +92,7 @@ const PromoCard: FunctionComponent< Props > = ( {
 						{ badgeComponent }
 					</ActionPanelTitle>
```

Comment:
Concern: should `titleLocation` not also apply to the `ActionPanelTitle`? If not, I would recommend we change the prop name to `titleComponentLocation`. If it should, then we should also add logic to handle that case as well.

Reply:
I'll rename, I think it's better to keep old title as it was. And if we want to modify it inject the title via props.